pkg/compose/compose_test.go

```diff
@@ -75,12 +79,34 @@ func TestComposeCompare(t *testing.T) {
 		composeBinsDir := t.TempDir()
 		compareDir = composeBinsDir
 		cockroachBin = filepath.Join(composeBinsDir, "cockroach")
-		err = copyBin(*flagCockroach, cockroachBin)
-		if err != nil {
+		libGeosDir = filepath.Join(composeBinsDir, "lib")
+		if err := copyBin(*flagCockroach, cockroachBin); err != nil {
 			t.Fatal(err)
 		}
-		err = copyBin(*flagCompare, filepath.Join(composeBinsDir, "compare.test"))
-		if err != nil {
+		// Copy files from flagLibGeosDir to libGeosDir.
+		if err := filepath.WalkDir(*flagLibGeosDir, func(path string, d os.DirEntry, err error) error {
```

Comment:
There are always exactly 2 GEOS libs, so you can just directly copy `libgeos.so` and `libgeos_c.so` instead of doing this.

Reply:
will do